### DISCUSSION TITLE
Jk BMS to Victron CCGX connection

### DISCUSSION THREAD

**Discussion by Dighy** [Louisvdw/dbus-serialbattery#519] [Q&A]
Hi everyone,
Please, help me to make the right serial connection from my JK BMS B2A8S20P to CCGX from Victron. Attached is my configuration that I have now. May I connect the USB cable to USB port of CCGX or need to connect to VE CAN through RJ45 cable.
If this is not ok, please, put the link here with proper stuff I need to buy :)
![IMG_20230408_152732](https://user-images.githubusercontent.com/127723834/230739552-07eb9c7f-9b0c-402e-a463-bf18e6472f1a.jpg)

**Comment by LecramRed** [comment]
Check off grid garage's video on youtube about this

**Reply by Dighy** [reply]
I did it, but there Andy is using "raspberry pi" device with USB port. I am using Color control GX device, and this one have a lot of ports. My dilemma is using usb port which as per user manual is only for GPS/driver update/router connection or to use VE can port which is for communication.
Thanks a lot for your time.

**Reply by LecramRed** [reply]
> I did it, but there Andy is using "raspberry pi" device with USB port. I am using Color control GX device, and this one have a lot of ports. My dilemma is using usb port which as per user manual is only for GPS/driver update/router connection or to use VE can port which is for communication. Thanks a lot for your time.

I have a Cerbo, too. You can use one of the two USB ports. Not the one that is meant for the display!

**Reply by Dighy** [reply]
Finally I will try one by one and see what happens, hope I will not damage anything. Problem is that USB ports are not isolated like VE CAN. ✌️
